flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/validate/FlinkSqlConformance.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.sql.parser.validate;
+
+import org.apache.calcite.sql.validate.SqlConformance;
+
+/** Sql conformance used for flink to set specific sql dialect parser. **/
+public enum FlinkSqlConformance implements SqlConformance {
+	/** Calcite's default SQL behavior. */
+	DEFAULT,
+
+	/** Conformance value that instructs Calcite to use SQL semantics
+	 * consistent with the Apache HIVE, but ignoring its more
+	 * inconvenient or controversial dicta. */
```

Comment:
"dicta" typo of "dialect" ?

Reply:
Controversial dicta means the grammar that are non-deterministic.